packages/client/package.json

```diff
@@ -1,9 +1,23 @@
 {
   "name": "@penumbra-zone/client",
-  "version": "1.0.0",
-  "description": "",
+  "version": "1.0.12",
+  "description": "Package for connecting to Prax extension",
+  "type": "module",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "scripts": {
+    "build": "pnpm clean && tsc && vite build",
```

Comment:
Conundrum, vite build is not idempotent for some reason. Exclude in tsconfig is not respected and therefore tries to re-bundle dist.

```
{
  "extends": "@penumbra-zone/tsconfig/base.json",
  "exclude": ["node_modules"],
  "compilerOptions": {
    "outDir": "dist"
  }
}

```

Reply:
calling `vite build` manually demonstrates that it is in fact idempotent. it's actually a `tsc` problem - the typescript compiler is refusing to clobber existing output

after some experimenting, it seems that `tsc` is unnecessary, and `vite build` by itself doesn't complain. if we're using `vite-plugin-dts` we should either not call `tsc` or use `noEmit` in tsconfig anyway

Reply:
I think we should keep tsc. It has managed to find type errors in a few packages that are otherwise going unnoticed.

Reply:
ok - then let's use tsconfig noEmit